src/main/java/org/example/catch_line/restaurant/model/entity/constant/DayOfWeeks.java

```diff
@@ -0,0 +1,37 @@
+package org.example.catch_line.restaurant.model.entity.constant;
+
+import java.time.DayOfWeek;
+
+public enum DayOfWeeks {
```

Comment:
```suggestion
public enum DayOfWeeks {
  MONDAY(DayOfWeek.MONDAY, "월요일"),
  TUESDAY(DayOfWeek.TUESDAY, "화요일"),
  // ... 다른 요일들
  
  private final DayOfWeek dayOfWeek;
  private final String description;
  
  // ... 생성자, 게터
  
  public static DayOfWeeks from(DayOfWeek dayOfWeek) {
    return Arrays.stream(values())
                .filter(e -> dayOfWeek == dayOfWeek)
                .findFirst()
                .orElse(null);
  }
}
```

이런식으로 DayOfWeek 를 내부 필드로 선언하면 from 메서드가 더 간략해지고, enum 이 추가되더라도 수정범위가 적어집니다.
물론 지금은 일주일을 나타내는거고, enum 이 추가될 여지가 없기 때문에 지금도 괜찮습니다. enum 을 이용할때 이런 방법도 있다 정도로 알아주세요.

개인적으로는 이름은 좀 변경하는걸 권하고 싶네요. java.time 에 있는 DayOfWeek 때문에 s 를 붙이는정도의 타협을 한 것 같은데 도메인을 모르는 제 3자가 코드를 볼때 헷갈릴 여지가 있어서요. `CatchLineDayOfWeek` 처럼 우리만의 DayOfWeek 라는걸 나타내는게 나중에 코드를 보는 사람들도 헷갈릴 여지가 줄어든다고 생각합니다 😃

Reply:
@LichKing-lee  헉...! 감사합니다! switch문을 써서 코드가 많이 지저분했었는데 알려주셔서 감사합니다!
stream을 잘 활용해보는 것이 좋은 것 같습니다! 알려주신대로 적용해보겠습니다!
리팩토링을 생각하고 있었는데 방법을 알려주셔서 또 하나 배울 수 있었습니다 😄